include/envoy/http/access_log.h

```diff
@@ -114,6 +114,16 @@ class RequestInfo {
    * Set whether the request is a health check request or not.
    */
   virtual void healthCheck(bool is_hc) PURE;
+
+  /**
+   * Set the downstream address.
+   */
+  virtual void setDownstreamAddress(const std::string& address) PURE;
```

Comment:
I've already mentioned this several times, but this setter should not be on the public interface. It doesn't make sense.